### PR TITLE
fix: render "Mint NFT" btn

### PR DESCRIPTION
**Pull request by swellander**
Fixes issue #170. See [screencast](https://drive.google.com/file/d/1ms5FsDVbRovW7HNlUGq9t7b4EK-OBDzz/view?usp=sharing)

### CODE REVIEW

File: src/pages/mint.tsx [hunk 1]
```diff
@@ -62,15 +62,17 @@ const MintPage: NextPage = () => {
...
+      contract?.isSaleActive()?.then((response) => {
+        setIsSaleActive(!!response)
+      })
+      contract?.totalSupply()?.then((response) => {
+        setTotalSupply(BigNumber.from(response).toNumber())
+      })
+      contract?.maxSupply()?.then((response) => {
+        setMaxSupply(BigNumber.from(response).toNumber())
+      })
+    }
```
Comment (swellander) [inline src/pages/mint.tsx:75]: These contract calls were throwing errors when not on mainnet

File: src/pages/mint.tsx [hunk 2]
```diff
@@ -244,27 +246,30 @@ const MintPage: NextPage = () => {
...
-                  >
-                    {translate('connectWallet')}
-                  </Button>
-                </VStack>
-              )}
+              {/* Not sold out, Minting not active, wallet not connected, on mainnet */}
+              {!isSoldOut &&
+                !isSaleActive &&
+                !walletConnected &&
+                isCorrectChainId && (
```
Comment (swellander) [inline src/pages/mint.tsx:253]: Line 253 has the only change here

**Comment by ctjlewis** [comment]
Thanks, great work!

**Comment by ctjlewis** [comment]
Thanks, great work!

**Pull request merged**